### PR TITLE
Enh/#1/conda env

### LINKED ISSUE #1: Module_1.2
Use Conda to set up a python development environment named `mle-dev` to run the script. Export the environments file as `env.yml` and commit it to the repository. @Senthil-Tiger

### PR DESCRIPTION
**Pull request by Senthil-Kumar-Uthiravel**
Close #1

### CODE REVIEW

File: env.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - sqlite=3.38.3=hc218d9a_0
+  - threadpoolctl=2.2.0=pyh0d69192_0
+  - tk=8.6.12=h1ccaba5_0
+  - tornado=6.1=py39h27cfd23_0
+  - tzdata=2022a=hda174b7_0
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - xz=5.2.5=h7f8727e_1
+  - zlib=1.2.12=h7f8727e_2
+  - zstd=1.5.2=ha4553b6_0
+prefix: /home/senthil_kumar/miniconda3/envs/mle-dev
```
Comment (sharma-sandy) [inline env.yml]: remove prefix

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+ - Random Forest
+
+## Steps performed
+ - We prepare and clean the data. We check and impute for missing values.
+ - Features are generated and the variables are checked for correlation.
+ - Multiple sampling techinuqies are evaluated. The data set is split into train and test.
+ - All the above said modelling techniques are tried and evaluated. The final metric used to evaluate is mean squared error.
+
+## To excute the script
+python <nonstandardcode.py>
```
Comment (sharma-sandy) [inline README.md]: use syntax highlighting in markdown

**Review by sharma-sandy — CHANGES_REQUESTED** [review]
Take a look at comments

File: README.md (hunk 2 above)
Comment (Senthil-Kumar-Uthiravel) [reply README.md]: I have updated it sir!! Do I need to create a new pull request again?

**Comment by Senthil-Kumar-Uthiravel** [comment]
> Take a look at comments

Changes made according to the comments. Could you look through and do the needful?

**Review by sharma-sandy — APPROVED** [review]

**Pull request merged**